modules/gapi/test/executor/tbb_executor_internal.cpp

```diff
@@ -0,0 +1,31 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+#include "executor/gapi_tbb_executor.cpp"
```

Comment:
.hpp file will not work, will it?

Reply:
not it will not as the tests use non-exported function

Reply:
Thanks for the explanation comment!